wger/core/management/commands/list-user-rest-api.py

```diff
@@ -0,0 +1,33 @@
+from django.core.management.base import (
+    BaseCommand, CommandError)
+from wger.core.models import (
+    ApiUserModel, User)
+from django_tabulate import tabulate_qs
+from tabulate import tabulate
+
+
+class Command(BaseCommand):
```

Comment:
Kindly use a more descriptive name to show what these commands are for. Say `CreateUserCommand`

Reply:
Its more of a must accordint to django docs that that should be Command

Reply:
Okay noted